src/sfizz/Region.cpp

```diff
@@ -349,9 +349,11 @@ bool sfz::Region::parseOpcode(const Opcode& rawOpcode)
             break;
         case hash("release"):
             trigger = SfzTrigger::release;
+            loopMode = SfzLoopMode::one_shot;
             break;
         case hash("release_key"):
             trigger = SfzTrigger::release_key;
+            loopMode = SfzLoopMode::one_shot;
             break;
```

Comment:
This behavior will not match to aria.
This is overriden by the `loop_mode` if present, regardless if happening before or after `trigger`.
For instance both of these will have `loop_sustain`.
```
<region> sample=sine440.wav pitch_keycenter=69 loop_mode=loop_sustain trigger=release_key
<region> sample=sine440.wav pitch_keycenter=69 trigger=release_key loop_mode=loop_sustain
```

Reply:
an loopmode is already an optional too! :)